### DISCUSSION TITLE
neo-tree background transparent

### DISCUSSION THREAD

**Discussion by Agus9217** [LazyVim/LazyVim#3021] [Q&A]
Hi I want to change neo-tree background transparent, but a don't know where is the configuration to do this, I'm using tokyonight whit the options transparent but neo tree does't work

**Comment by dpetka2001** [comment]
Please take a look [here](https://github.com/LazyVim/LazyVim/discussions/116#discussioncomment-4763452).

**Reply by Agus9217** [reply]
I did that, but doesn't work, I take the same code

**Reply by dpetka2001** [reply]
You also have to configure your terminal to be transparent.

**Reply by Agus9217** [reply]
Yes I know that, all the UI is transparent, when toggle neo-tree, neo-tree background is not transparent, all the rest work prefectly

**Reply by dpetka2001** [reply]
I have exactly this code 
```lua
  {
    "folke/tokyonight.nvim",
    opts = {
      transparent = true,
      styles = {
        sidebars = "transparent",
        floats = "transparent",
      },
    },
  },
```
and everything is transparent. You're doing something wrong in your configuration.

**Reply by Agus9217** [reply]
I'm going to review my code again
